easybuild/easyconfigs/n/NewHybrids/NewHybrids-1.1_Beta3-GCC-10.2.0.eb

```diff
@@ -0,0 +1,45 @@
+# Author: Denis Krišťák (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'NewHybrids'
+version = '1.1_Beta3'
+
+homepage = 'http://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm#NewHybs'
+description = """This implements a Gibbs sampler to estimate the posterior probability that genetically sampled
+individuals fall into each of a set of user-defined hybrid categories."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+sources = [{
+    'filename': SOURCE_TAR_GZ,
+    'git_config': {
+        'url': 'https://github.com/eriqande',
+        'repo_name': 'newhybrids',
+        'commit': '6fc8fd9ca6e64ee0be08af12ec8bfe73a7de00cc',
+        'recursive': True,
+    }
+}]
+
+skipsteps = ['configure']
+
+build_cmd = './Compile-with-no-gui-Linux.sh'
+
+files_to_copy = ['*']
```

Comment:
I think using the `Binary` easyblock is better here?

Use `install_cmd` to run `./Compile-with-no-gui-Linux.sh'` and then copy the binary with `cp -a` into `bin` subdir in installation directory?

Reply:
are we sure that user doesnt need other binaries from `builddir` (for mac and linux)?